.github/workflows/smoke.yml

```diff
@@ -33,9 +33,6 @@ jobs:
       - name: Setup
         run: npm i -g @antfu/ni
 
-      - name: Setup PNPM
-        uses: pnpm/action-setup@v2
```

Comment:
Why do we remove this?

Reply:
This seems to be unused. Also, test.ci and release.ci don't have this.